packages/domain/lib/use_case/log_analytics_event_use_case.dart

```diff
@@ -0,0 +1,11 @@
+import 'package:domain/services/analytics_service.dart';
+import 'package:domain/use_case/use_case.dart';
+
+class LogAnalyticsEventUseCase extends UseCaseParams<String, Future<void>> {
```

Comment:
mb LogAnaliticsOnTapUseCase for example or some another name to tell what it exactly serves for?